src/Registry/registry_instance.jl

```diff
@@ -15,15 +15,15 @@ function to_tar_path_format(file::AbstractString)
 end
 
 # See loading.jl
-const TOML_CACHE = Base.TOMLCache(TOML.Parser(), Dict{String, Dict{String, Any}}())
+const TOML_CACHE = Base.TOMLCache(Base.TOML.Parser(), Dict{String, Dict{String, Any}}())
 const TOML_LOCK = ReentrantLock()
 _parsefile(toml_file::AbstractString) = Base.parsed_toml(toml_file, TOML_CACHE, TOML_LOCK)
 function parsefile(in_memory_registry::Union{Dict, Nothing}, folder::AbstractString, file::AbstractString)
     if in_memory_registry === nothing
         return _parsefile(joinpath(folder, file))
     else
         content = in_memory_registry[to_tar_path_format(file)]
-        return TOML.Internals.parse(TOML.Parser(content; filepath=file))
+        return TOML.Internals.parse(Base.TOML.Parser(content; filepath=file))
```

Comment:
Seems odd to use `TOML.Internals` here but pass `Base.TOML` as an argument. Maybe instead change `using TOML` to `using Base.TOML` at the top of the file?

Reply:
bump?